src/main/java/core/basesyntax/strategy/OperationService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationService {
+    void proceed(FruitTransaction fruitTransaction);
```

Comment:
```suggestion
    void process(FruitTransaction fruitTransaction);
```
or something else but not proceed